pyfar/dsp/filter/gammatone.py

```diff
@@ -0,0 +1,581 @@
+import numpy as np
+import scipy.signal as sgn
+from copy import deepcopy
+from deepdiff import DeepDiff
+import pyfar as pf
+
+
+class GammatoneBands():
```

Comment:
Why not call it what it is?
```suggestion
class GammatoneFilterbank():
```